data_storage/export_format.md

```diff
@@ -1,22 +1,28 @@
 Export Format
 -------------
 
-- **5.1.1** ![](/badge/req.png) A Toxcore data file saved in the root of the client's
-  working directory, with the file name `<profile_name>.tox`
+- **5.1.1** ![](/badge/req.png) The client will export files relative to a folder called
+  the export folder, which must be in a location accessible to the user.
 
-- **5.1.2** ![](/badge/req.png) If the client supports user avatars, they will be stored
-  under the `avatars/` directory in the root of the client's working directory.
+- **5.1.2** ![](/badge/rec.png) The user should be able to choose the location
+  of the export folder.
 
-- **5.1.3** ![](/badge/req.png) Avatars are named according to the following format:
+- **5.1.3** ![](/badge/req.png) A Toxcore data file saved in the root of the
+  export folder, with the file name `<profile_name>.tox`.
+
+- **5.1.4** ![](/badge/req.png) If the client supports user avatars, they will be stored
+  under the `avatars/` directory in the root of the export folder.
+
+- **5.1.5** ![](/badge/req.png) Avatars are named according to the following format:
   `PK.png`, where PK is the 64-character hexadecimal public key of its owner.
 
 - An example file hierarchy:
 
   ```
   +-----------------------------------------------------------------------------------+
-  |   / (client working dir)                                                          |
+  |   / (export folder)                                                               |
   |    |                                                                              |
-  |    +-- tox_save.tox                                                               |
+  |    +-- profile_name.tox                                                           |
```

Comment:
This is an arbitrary name meaningful to (and chosen by) the user, unique per Tox profile, it (obviously) has to be a valid filename on the user's filesystem.

It may be derived from, or identical to, the Tox Nickname (as returned by tox_self_get_name) as long as it's recognizable by the user.